src/c++/perf_analyzer/concurrency_manager.cc

```diff
@@ -128,11 +128,9 @@ ConcurrencyManager::ReconfigThreads(const size_t concurrent_request_count)
     threads_config_.emplace_back(
         new ConcurrencyWorker::ThreadConfig(threads_config_.size()));
 
-    auto worker = MakeWorker();
+    auto worker = MakeWorker(threads_stat_.back(), threads_config_.back());
```

Comment:
Can these parameters just be moved inside MakeWorker? They're already data members.

Reply:
They are, but that would couple the two. It would assume that MakeWorker was only ever called in a loop where the last member of threads_stat_ and threads_config_ were the ones for the worker.